ci-operator/step-registry/cucushift/installer/rehearse/azure/aro/pvt/cucushift-installer-rehearse-azure-aro-pvt-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-aro-pvt
+  steps: 
+    pre:
+    - ref: azure-provision-resourcegroup
+    - ref: aro-provision-vnet
+    - chain: azure-provision-bastionhost
+    - ref: proxy-config-generate
+    - ref: aro-provision-cluster
+    - ref: ipi-install-rbac
+    - ref: openshift-cluster-bot-rbac
+    - ref: enable-qe-catalogsource
+    post:
+    - ref: aro-deprovision
```

Comment:
If this workflow is used for private cluster install, suggest to set the default env vars here.
```
    env:
      ARO_API_SERVER_VISIBILITY: Private
      ARO_INGRESS_VISIBILITY: Private
      ARO_OUTBOUND_TYPE: UserDefinedRouting
```